### PR TITLE
Some plushie grammar fixes

### PR DESCRIPTION
**Pull request by kriskog**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Skoglol
spellcheck: Fixed Bubblegum plushie attack verb, Ratvar and Bubblegum plushie capitalization.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

### CODE REVIEW

**Comment by subject217** [comment]
speedmerge please

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -372,14 +372,14 @@
 	squeak_override = list('sound/weapons/bite.ogg'=1)
 
 /obj/item/toy/plush/bubbleplush
-	name = "bubblegum plushie"
+	name = "\improper Bubblegum plushie"
 	desc = "The friendly red demon that gives good miners gifts."
 	icon_state = "bubbleplush"
-	attack_verb = list("rends")
+	attack_verb = list("rended")
```
[the remark this answers is not in the document]
Comment (kriskog) [reply code/game/objects/items/plushes.dm]: Yeah, they are all supposed to be past tense. Did you see any other mistakes?

[the remark this answers is not in the document]
Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/plushes.dm]: Not on this file but i'll be on the lookout.

[the remark this answers is not in the document]
Comment (kriskog) [reply code/game/objects/items/plushes.dm]: To be clear: "<user> has <attack_verb> <target> in the <location> with <plushie>.
This is just eating verbs all over again.

Comment (kriskog) [inline code/game/objects/items/plushes.dm]: "rended" or "rent"? Both are valid, what do people like more?

Comment (Time-Green) [reply code/game/objects/items/plushes.dm]: definitely rent

Comment (kriskog) [reply code/game/objects/items/plushes.dm]: Yeah, that was my preference too

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**